extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -592,43 +587,38 @@ function axWFactorySaveVariable() {
 						$return = Wikia::successmsg( "Parse OK, variable saved." );
 					}
 				}
-				ob_end_clean(); #--- puts parse error to /dev/null
+				#--- puts parse error to /dev/null
+				ob_end_clean();
 		}
 
-                if( empty( $error ) ) {
-                    $varInfo = WikiFactory::getVarById($cv_id, $city_id);
-                    if($varInfo->cv_is_unique) {
-                        $wikis = WikiFactory::getCityIDsFromVarValue( $cv_id, $cv_value, '=');
-                        $count = count($wikis);
-                        if( (($count == 1) && ($wikis[0] != $city_id)) || ($count > 1) ) {
-                            $return = Wikia::errormsg( "Value of this variable need to be unique." );
-                            $error++;
-                        }
-                    }
-                }
-
-		wfRunHooks('WikiFactoryVarSave::AfterErrorDetection',array($cv_id,$city_id,$cv_name,$cv_value,&$return,&$error));
+		if ( empty( $error ) ) {
+			$varInfo = WikiFactory::getVarById( $cv_id, $city_id );
+			if ( $varInfo->cv_is_unique ) {
+				$wikis = WikiFactory::getCityIDsFromVarValue( $cv_id, $cv_value, '=');
+				$count = count( $wikis );
+				if ( ( $count == 1 && $wikis[0] != $city_id ) || $count > 1 ) {
+					$return = Wikia::errormsg( "Value of this variable needs to be unique." );
+					$error++;
+				}
+			}
+		}
 
+		wfRunHooks(
+			'WikiFactoryVarSave::AfterErrorDetection',
+			[ $cv_id, $city_id, $cv_name, $cv_value, &$return, &$error ]
+		);
 
 		# Save to DB, but only if no errors occurred
 		if ( empty( $error ) ) {
-			if( ! WikiFactory::setVarByID( $cv_id, $city_id, $cv_value, $reason ) ) {
+			if ( ! WikiFactory::setVarById( $cv_id, $city_id, $cv_value, $reason ) ) {
 				$error++;
 				$return = Wikia::errormsg( "Variable not saved because of problems with database. Try again." );
 			} else {
-				/* the one "set" that used this is now disabled, so disabling the call until needed again
-				$tied = WikiFactory::getTiedVariables( $cv_name );
-				if( $tied ) {
-					$return .= Wikia::successmsg(
-						" This variable is tied with others. Check: ". implode(", ", $tied )
-					);
-				}
-				*/
```

Comment:
If the above is functionality that's really needed later, it can be added back later.